kinds/preferences/preferences_kind.cue

```diff
@@ -44,5 +47,16 @@ lineage: schemas: [{
 			performance?: {}
 			functional?: {}
 		} @cuetsy(kind="interface")
+
+		#NavbarPreference: {
+			savedItems: [...#NavLink]
+		} @cuetsy(kind="interface")
+
+		#NavLink: {
```

Comment:
We have `NavLinkDTO` type for the navModel, so maybe this'd be renamed to `PreferenceNavLink` or similar to clearly show that this is a nav link from preferences and not a regular nav item?